modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/validators.js

```diff
@@ -0,0 +1,366 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const ERROR_OBJECT_OF_TYPE = 'Expected object of one type.';
+const ERROR_ONE_OF = 'Expected one of the following values:';
+const ERROR_ONE_OF_TYPE = 'Expected one of given types.';
+
+function _getFunctionName(fn) {
```

Comment:
Remove leading underscore — this is private to the module no matter what it is called so there is no need to worry about somebody accidentally accessing something they are not supposed to. Same for any other function following the same pattern in here.